<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailAccountDialog</class>
 <widget class="QWizard" name="MailAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mail Accounts</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <widget class="QWizardPage" name="profilesDirPage">
   <property name="title">
    <string>Thunderbird Profiles Directory</string>
   </property>
   <property name="subTitle">
    <string>Select the directory that contains the Thunderbird profiles.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">MailAccountDialog::PageId::profilesDirPage</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="tbProfilesPathEdit">
      <property name="placeholderText">
       <string>Thunderbird Profiles Directory</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tbProfilesBrowseButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="profilePage">
   <property name="title">
    <string>Thunderbird Profile</string>
   </property>
   <property name="subTitle">
    <string>Select theThunderbird profile, that contains the mail account which you want to monitor.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">MailAccountDialog::PageId::profilePage</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Selected Profile:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="profileSelector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="accountsPage">
   <property name="title">
    <string>Select Accounts</string>
   </property>
   <property name="subTitle">
    <string>Select the mail accounts you want to monitor.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">MailAccountDialog::PageId::accountsPage</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="accountsList">
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>300</number>
      </attribute>
      <column>
       <property name="text">
        <string>Email Folder</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notification Color</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>If you monitor multiple folders, the default notification color is used to show the sum of all unread mails.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
